Prace_domowe/Praca_domowa2/Jakub_Foltyn/PD2.ipynb

```diff
@@ -0,0 +1,1178 @@
+{
```

Comment:
<p>Aby móc lepiej porównywać wyniki pomiędzy testem 1 i 2, dobrym rozwiązaniem byłoby gdyby random_indexes_1 było równe random_indexes_2.</p><p> </p><p>Błąd należałoby wyliczyć tylko dla wartości imputowanych, bo gdy wyliczamy dla całej kolumny wynik jest zakłamany, ponieważ we wzorze jest wyliczana średnia, przez co liczność ma znaczenie.</p><p>Warto byłoby policzyć pierwiastek z błędu średniokwadratowego - można na przykład użyć parametru squared.</p><p><u>Prośba o zmodyfikowanie sposobu wyliczania błędu o powyższe dwa aspekty.</u></p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/mini-pw/2021L-WUM/pull/63/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='21'/>